.github/workflows/sdk-build.yml

```diff
@@ -8,14 +8,18 @@ on: [push, pull_request]
 jobs:
   build:
     if: (github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository) && !contains(github.event.head_commit.message, '[ci skip]')
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [3.7, 3.8, 3.9, 3.10, 3.11]
```

Comment:
The Python versions need to be wrapped in quotes. Currently `3.10` is being treated as `3.1` since YAML thinks it is a floating point number that should be rounded.